### PR TITLE
45 hashtag search + 40 navbar 

### LINKED ISSUE #40: Navbar updates
Navbar should have a home/posts button that leads to the page with all posts. The navbar and its buttons should also be a bit larger

### LINKED ISSUE #45: Hashtag search
Users should be able to click on hashtags, or search them in the searchbar, and a list of posts with that hashtag should be returned.

### PR DESCRIPTION
**Pull request by melleg**
Closes #40 and closes #45

### CODE REVIEW

File: backend/src/main/java/nl/itvitae/twitbook/post/PostController.java [hunk 1]
```diff
@@ -84,6 +87,19 @@ public ResponseEntity<?> getAllByUsername(@PathVariable String username,
     return ResponseEntity.ok(posts.map(p -> getPostDTO(p, user)));
   }
 
+  @GetMapping("by-hashtag/{hashtagText}")
+  public ResponseEntity<?> queryByHashtag(@PathVariable String hashtagText,
+      @AuthenticationPrincipal User user, Pageable pageable) {
+
+    Optional<Hashtag> hashtag = hashtagService.findHashtagByText(hashtagText);
+    if(hashtag.isEmpty()) return ResponseEntity.ok(getPageable(pageable)); // Return empty page
+
```
Comment (NolKeereweer) [inline backend/src/main/java/nl/itvitae/twitbook/post/PostController.java:96]: Few whitespaces too many?

File: backend/src/main/java/nl/itvitae/twitbook/post/PostRepository.java
Comment (NolKeereweer) [inline backend/src/main/java/nl/itvitae/twitbook/post/PostRepository.java:1]: Is there a reason for having no whitespaces?

File: backend/src/main/java/nl/itvitae/twitbook/user/UserRepository.java
Comment (NolKeereweer) [inline backend/src/main/java/nl/itvitae/twitbook/user/UserRepository.java:1]: Again some inconsistent whitespacing

**Review by NolKeereweer — APPROVED** [review]
Perfectly fine apart from some small whitespace inconsistencies.

**Review by mio-xenoblade — APPROVED** [review]

**Pull request merged**